db/migrate/20180519050412_create_lessons.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLessons < ActiveRecord::Migration[5.1]
+  def change
+    create_table :lessons do |t|
+      t.integer :user_id
+      t.integer :category_id
```

Comment:
@Masa08 Use `t.references :category, foreign_key: true`, to set foreign keys. Please check other migrations, and change all foreign keys to this format.
See here: http://guides.rubyonrails.org/association_basics.html